src/renderer/containers/CustomDataTable/selectors.tsx

```diff
@@ -87,6 +88,16 @@ const DEFAULT_COLUMNS: CustomColumn[] = [
 ];
 
 export const PLATE_RELATED_COLUMNS: CustomColumn[] = [
+  {
+    accessor: AnnotationName.PROGRAM,
+    Cell: ProgramCell,
+    // This description was pulled from LK 04/17/25
+    description: "Name of project or program a dataset can be used for.",
+    width: getColumnWidthForType(
+      AnnotationName.PROGRAM,
+      ColumnType.LOOKUP
```

Comment:
I checked out this branch, and when I tried to select a value for the "Program" annotation (which does show up, which is good) all I got was an empty dropdown that said "no data". Have you managed to get the dropdown to populate properly?

For reference, here are cases where files are annotated with the "Program" annotation already:
https://stg-aics.corp.alleninstitute.org/labkey/query/AICS/executeQuery.view?schemaName=filemetadata&query.queryName=CustomAnnotationJunction&query.AnnotationId%2FName~contains=Program